timur/lib/commands.rb

```diff
@@ -52,7 +52,7 @@ def execute(version: nil)
 
     def setup(config)
       super
-      Timur.instance.setup_db
+      Timur.instance.setup_db(false)
```

Comment:
Does this throw an exception? I thought the [`setup_db` method doesn't take arguments](https://github.com/mountetna/monoetna/blob/master/timur/lib/timur.rb#L7)?

Reply:
Hmm yeah I don't get this either - won't this not load the models?

Reply:
@coleshaw I added the argument to make this work.

Without it, tests fail to migrate.  It seems with the ruby upgrade the import ordering changes?  But on the migrate command the models should not be loaded, since we're likely creating tables for things before they exist.

Anyways, it is required now or tests fails in CI when importing models that aren't migrated yet.

Reply:
:+1: I didn't see the `timur.rb` change when I made my comment, makes sense now.